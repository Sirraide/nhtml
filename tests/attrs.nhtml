svg[width=1.5rem, height=1.2rem, version=1.1, viewBox=0 0 1.5rem 1.2rem, xmlns=http://www.w3.org/2000/svg]#hamburger {
    rect[width=1.5rem, height=.2rem]
    rect[width=1.5rem, height=.2rem, y=.5rem]
    rect[width=1.5rem, height=.2rem, y=1rem]
}