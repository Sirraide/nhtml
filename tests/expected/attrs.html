<svg id='hamburger' height='1.2rem' version='1.1' viewbox='0 0 1.5rem 1.2rem' width='1.5rem' xmlns='http://www.w3.org/2000/svg'>
    <rect height='.2rem' width='1.5rem'></rect>
    <rect height='.2rem' width='1.5rem' y='.5rem'></rect>
    <rect height='.2rem' width='1.5rem' y='1rem'></rect>
</svg>
